dist_tables/dml.rst

```diff
@@ -38,6 +38,70 @@ For example:
 
     If COPY fails to open a connection for a shard placement then it behaves in the same way as INSERT, namely to mark the placement(s) as inactive unless there are no more active placements. If any other failure occurs after connecting, the transaction is rolled back and thus no metadata changes are made.
 
+INSERT INTO ... SELECT
+$$$$$$$$$$$$$$$$$$$$$$
+
+Citus is commonly used to scale out event data pipelines and populate real-time dashboards. These applications require fast queries across predefined aggregations of incoming data. To achieve sufficient speed with high-volume data, it helps to pre-aggregated data. This is called "rolling up" the data and it avoids the cost of processing raw data at run-time. As an extra benefit rolling up timeseries data into hourly or daily statistics can also save space. When the full details of old data are no longer needed and aggregates suffice then old data may be deleted.
+
+It is important to be able to update rollups *incrementally*, that is, without re-scanning all data. Both the rolling up and updating can be accomplished with :code:`INSERT INTO ... SELECT`, whereby the results of a select query are inserted into a table. A full reference for this feature is available in the PostgreSQL `docs <https://www.postgresql.org/docs/current/static/sql-insert.html>`_. It works well on Citus because it is parallelizable under certain conditions. We'll see more about that later.
+
+First some basics. Insert into select uses the results of a select query to populate a table. For instance consider a simple select statement which counts the number of web page views per page per day.
```

Comment:
Small. We use "insert into select" several times in this section.

Is it worth standardizing how we write it down? Insert Into Select, ``INSERT INTO SELECT``, etc.?